Resources/config/controller.xml

```diff
@@ -5,18 +5,14 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="HWI\Bundle\OAuthBundle\Controller\ConnectController" public="true">
+        <service id="HWI\Bundle\OAuthBundle\Controller\ConnectController" public="true" autowire="true">
```

Comment:
we should not use `autowire` here. Its not a good practice to do this on third-party-bundles. Also see https://symfony.com/doc/current/bundles/best_practices.html#services

Reply:
ah ok